airgun/entities/host.py

```diff
@@ -327,6 +328,10 @@ def step(self, *args, **kwargs):
         entity_name = kwargs.get('entity_name')
         self.parent.search(entity_name)
         self.parent.table.row(name=entity_name)['Name'].widget.click()
+        host_view = NewHostDetailsView(self.parent.browser)
```

Comment:
VIEW for this class is still set to `HostDetailsView`, I don't think your change is fully addressing using the new view by default.

Reply:
This change should not use the New host details UI, the only change here is that default navigation would be to the new UI where we need to click on the `Legacy UI` button, so the rest of the view will actually still be the `HostDetailsView`

Reply:
So we don't want to test the NewHostDetailsView yet? We are just using this to workaround for automation until that's ready?

Reply:
This is workaround for those test that we already have implemented for old UI. I have separate PRs for the new host details page. As the UI changed so much it does not make sense to rewrite old testes.